pymerc/game/recipe_manager.py

```diff
@@ -0,0 +1,39 @@
+import asyncio
+from pymerc.client import Client
+from pymerc.game.recipe import Recipe
+
+
+class RecipeManager:
```

Comment:
I think this class might be unnecessary. My concern is that using it actually ends up calling `get_recipes()` on the static API for each recipe, which is a very expensive call. Ideally, we should only load once, if possible. 

A way I was already thinking of solving this is by using caching. Adding [this decorator](https://docs.python.org/dev/library/functools.html#functools.lru_cache) to the static API functions will cache the results after the first call to them. Since the data really shouldn't change, I think this is safe.